### PR TITLE
:arrow_up: Avoid deprecated dependencies

### LINKED ISSUE #743: `rush check` shows mismatching dependencies
image

### PR DESCRIPTION
**Pull request by MikuroXina**
Resolves #743 and replaces deprecated dependencies with newer.

### Changelog

Fixed a problem where the project contains multiple versions of dependencies, upgraded CircleCI build image, upgraded and migrated from deprecated dependencies.

### CODE REVIEW

**Comment by MikuroXina** [comment]
@dtinth Could you review my PR, please?

**Review by dtinth — APPROVED** [review]
@MikuroXina Hi — many thanks for the changes. I am not currently active on this project, so I will have to get back to it later, but I appreciate it! This should go with the next update. :+1:

**Review by dtinth — APPROVED** [review]
Looking good! Just one comment

File: bemuse/package.json [hunk 1]
```diff
@@ -168,7 +165,10 @@
     "whatwg-fetch": "^1.1.1",
     "downshift": "~6.1.7",
     "@radix-ui/react-alert-dialog": "~0.1.5",
-    "fuzzysort": "~1.1.4"
+    "fuzzysort": "~1.1.4",
+    "fancy-log": "~2.0.0",
+    "@descript/web-audio-js": "~0.15.3-patched.5",
```
[the remark this answers is not in the document]
Comment (MikuroXina) [reply bemuse/package.json]: I just mistook to rebase about them, sorry. Fixed on e3e438b0.

**Pull request merged**